### PR TITLE
Add the UI Toolkit to the doc

### PR DESCRIPTION
**Pull request by andy-armstrong**
This commit introduces WebPack to bundle all the assets together so that we don't have complicated URL path issues in the Pattern Library doc site.

I'm not yet including any UI Toolkit classes as there is an issue with the multiple versions of JQuery that I need to resolve first. Hopefully this still might be useful enough for folks to start using.

@AlasdairSwan @dsjen @clrux This is for the hackathon so no need to review yet, but feel free to provide feedback.

### CODE REVIEW

**Comment by edx-webhook** [comment]
## PR Cover Letter
- [ ] A detailed description of the changes in the body of the PR
- [ ] If available, a link to the relevant JIRA ticket in the body of the PR
  - `[FEDX-XXXX](https://openedx.atlassian.net/browse/FEDX-XXXX)`

## Testing
- [ ] i18n
- [ ] Responsive design
- [ ] RTL

## Reviewers
If you've been tagged for review, please check your corresponding box once you've given the :+1:.
- [ ] Code review: @Reviewer1
- [ ] Code review: @Reviewer2
- [ ] Documentation review: @...
- [ ] UX review: @...
- [ ] Accessibility review: @...
- [ ] Product review: @...

FYI: Tag anyone who might be interested in this PR here.

## Post-review
- [ ] Squash commits
- [ ] Publish packages/doc site

**Comment by andy-armstrong** [comment]
@AlasdairSwan @dsjen This is now ready for review. It picks up the new 0.3.0 version of the UI Toolkit, and it successfully bundles the disclosure view into the page.

**Comment by AlasdairSwan** [comment]
Awesome :+1:

File: webpack.config.js [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+                'edx-ui-toolkit': path.resolve(__dirname, 'node_modules/edx-ui-toolkit/src')
+            }
+        },
+        plugins: [
+            new webpack.ProvidePlugin({
+                $: 'jquery'
+            }),
+            new webpack.IgnorePlugin(/^(config.js)$/)
+        ],
+        debug: true,
```
Comment (dsjen) [inline webpack.config.js]: In the future, I guess we'd want to switch this field for prod deployments.

**Comment by dsjen** [comment]
I ran this locally, and it works great! :+1:

**Pull request merged**